src/TreeMap/App.svelte

```diff
@@ -1,590 +1,629 @@
+          },
         });
 
-        map.on("load", () => {
-            fetchContributionsData.then((contributions) => {
-                if (contributions.length > 0) {
-                    contributions.map((contribution) => {
-                        if (contribution.geometry) {
-                            var el = document.createElement("div");
-                            var treeCount = document.createTextNode(
-                                contribution.properties.treeCount
-                            );
-
-                            el.appendChild(treeCount);
-
-                            var svgNS = "http://www.w3.org/2000/svg";
-                            var tree = document.createElementNS(svgNS, "svg");
-
-                            tree.setAttributeNS(null, "width", 10.5);
-                            tree.setAttributeNS(null, "height", 12.598);
-                            tree.setAttributeNS(
-                                null,
-                                "viewBox",
-                                "0 0 10.5 12.598"
-                            );
-
-                            var pathNS = "http://www.w3.org/2000/svg";
-                            var path = document.createElementNS(pathNS, "path");
-                            path.setAttributeNS(
-                                null,
-                                "d",
-                                "M9,15.1V12.235a2.9,2.9,0,0,1-1,.175,3.033,3.033,0,0,1-2.093-5.2,3.055,3.055,0,0,1-.24-1.19,3,3,0,0,1,5.5-1.68h.167a3.7,3.7,0,1,1-1,7.252v3.5Z"
-                            );
-                            path.setAttributeNS(
-                                null,
-                                "transform",
-                                "translate(-4.75 -2.75)"
-                            );
-                            path.setAttributeNS(null, "fill", primaryColor);
-                            path.setAttributeNS(null, "stroke", "#fff");
-                            path.setAttributeNS(null, "stroke-width", 0.5);
-                            tree.appendChild(path);
-                            el.appendChild(tree);
-
-                            el.className = "marker";
-
-                            // make a marker for each feature and add to the map
-                            new mapboxgl.Marker(el, {
-                                anchor: "bottom",
-                            })
-                                .setLngLat(contribution.geometry.coordinates)
-                                .addTo(map);
-                        }
-                    });
-                }
-            });
+        map.addLayer({
+          id: "contrib",
+          type: "circle",
+          source: "contributions",
+          filter: ["!", ["has", "point_count"]],
+          paint: {
+            "circle-color": "#68B030",
+            "circle-radius": 20,
+            "circle-stroke-width": 1,
+            "circle-stroke-color": "#fff",
```

Comment:
I would rather prefer to mark the cluster with an extra border instead of the single contribution as done here. And I think a white border does not work well in the `light` theme.

Reply:
I changed it now (also merging the dark mode styles from develop). Are you ok with that?

Reply:
sure, I didn't see this before pushing

Reply:
reverted my changes